jupyterbook/_config.yml

```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url: "https://mybinder.org"
+  colab_url: "https://colab.research.google.com"
+  thebe: true
+
+
+#######################################################################################
+# Execution settings
+execute:
+  execute_notebooks           : cache
+
+#######################################################################################
+# HTML-specific settings
+html:
+  home_page_in_navbar         : false
+
+
+# #######################################################################################
+# Interact link settings #
+notebook_interface            : "notebook"
+
+#######################################################################################
+# Launch button settings
+#repository:
+#  url                         : https://github.com/
+
+#  use_repository_button     : true  # Whether to add a link to your repository button
+#  use_issues_button         : true  # Whether to add an "open an issue" button
+latex:
+  latex_engine                : "xelatex"
```

Comment:
Probably not necessary but I guess it only runs if you want a pdf version, right?

Reply:
There is a download option that is available for each of the notebooks. The `latex_engine` option is for the same.